docker.md

```diff
@@ -0,0 +1,23 @@
+# To build the image
```

Comment:
If you’re already going to do this, why not also add a quick link to a guide/set of instructions for how to install Docker.

Reply:
It's a work in progress. didn't get to that yet.

Reply:
I will. didn't get to it yet.